backend/models/post.ts

```diff
@@ -1,7 +1,8 @@
-import { Schema, model } from 'mongoose';
+import { Model, Schema, model } from 'mongoose';
+import { IPost } from '../types/post-Type';
 
 const postSchema = new Schema({
-  authorName: String,
+  authorfName: String,
   title: String,
```

Comment:
Why fName? keep it same please

Reply:
sorry, i didnt know that. some errors by mistakely done,i'll fix